### ISSUE TITLE
Xception: Is there a need for batch_norm to be deactivated during evaluation?

### ISSUE THREAD

**Issue by kwotsin** [fchollet/deep-learning-models#62]
In the inception models (for TF-slim implementations), it seems that batch norm is deactivated once the model is sent for evaluation. Does Keras have this functionality as well, and is this option available for the Xception model?

Is there a need for deactivating the batch_norm during evaluation?

Thank you! :D
